### ISSUE TITLE
Once the Evaluator has fitted, Can I export the best ensemble directly?

### ISSUE THREAD

**Issue by jackmiemie** [flennerhag/mlens#58]
Hi, mlens is a great tool! It saves my life and I use it to do kaggle competition.

The evaluator is a nice tool for model selection of meta-layer, but, evaluating an ensemble network consumes a lot of time. Can I export the best ensemble directly once the evaluator has fitted? 

Now my work flow using mlens is like:
1) build the ensemble transformer
2) build the evaluator for meta-layer selection
3) fit the evaluator, and find the best meta-layer for the ensemble
4) build the ensemble with best meta-layer, and fit the ensemble again.

Step 4) is a bit duplicated and has double time expenses.

Is there anyway to avoid step 4) ? Am I using mlens correctly ?

**Comment by flennerhag** [comment]
Hi there, 

Thanks, glad to hear it helps! 

Good question. First, a comment. The ensemble transformer is fitted on *folds* and never all data. Hence after the model selection step, you might still want to fit the ensemble on all data. Same goes for the meta learner.

Now, there might be reasons you want to just grab on of the fitted transformers. In this case, the ``preprocess`` (or maybe ``preprocess_``) attribute on the ``Evaluator`` contains fitted transformers and you could grab one of those. I think fitted estimators aren't actually stored currently so the meta learner is problematic. The ``0.2.0`` version is coming out any day now and then the estimators will be accessible as well.

For exporting an ensemble transformer into a proper ensemble class, do

```python
ensemble = Sequential(layers=ensemble_transformer.layers, ...)
```

You'd then need to add the meta learner via the ``add`` method. You might actually be able to use ``ensemble = SomeClass(**ensemble_transformer.get_params(deep=False))``. If so please shout out, would be nice to document.

Note that in ``0.2.0`` the ensemble transformer will likely be depricated (but still available) in favor of using the ensemble classes directly together with a ``model_selection`` argument (or something such, haven't decided on a name yet).

**Comment by flennerhag** [comment]
Closed as of #67.
